.irb_config

```diff
@@ -0,0 +1,8 @@
+# Note that we use a non-standard config file name to reduce
+# name clashes with other IRB config files like `.irbrc`.
+
+require 'irb/completion'
+
+IRB.conf[:SAVE_HISTORY] = 100
+IRB.conf[:HISTORY_FILE] = "#{__dir__}/.irb_history"
```

Comment:
Since I haven't been able to get this working on system Ruby it'd be nice to skip these lines in that case. What would be the best way to see if we're using system Ruby vs. installed Ruby or portable Ruby?

Reply:
It will write to the file but not read from it but shouldn't cause any other problems.